Data/Scripts/OreDetectorReforged/Detector/SearchTask.cs

```diff
@@ -7,23 +7,26 @@ namespace OreDetectorReforged.Detector
     class SearchTask
     {
         public BoundingSphereD area;
+        public Vector3D center; 
         public string minedOre;
         public Func<Vector3D, bool> resultCb;
         public Action finishCb;
         public List<IDetectorPage> pages;
 
-        public SearchTask(BoundingSphereD area, string minedOre, Func<Vector3D, bool> resultCb, Action finishCb)
+        public SearchTask(BoundingSphereD area, Vector3D center, string minedOre, Func<Vector3D, bool> resultCb, Action finishCb)
```

Comment:
This confuses the principal function is no longer a nearest neighbor search. Instead, you start searching around center and if the nearest is not inside area you just break and find nothing. And if you didn't break it would be slow.

Reply:
I didn't change function itself. center is used to calculate distance from marker. area.center is still used for searching.

Reply:
I got it. These changes are irrelevant then) I misread how GPS markers work.